doc/user/pim.rst

```diff
@@ -93,30 +93,30 @@ Certain signals have special meanings to *pimd*.
    down. This command is vrf aware, to configure for a vrf, enter the vrf
    submode.
 
-.. clicmd:: ip pim join-prune-interval (5-600)
+.. clicmd:: ip pim join-prune-interval (1-65535)
 
    Modify the join/prune interval that pim uses to the new value. Time is
    specified in seconds. This command is vrf aware, to configure for a vrf,
    enter the vrf submode.  The default time is 60 seconds.  If you enter
    a value smaller than 60 seconds be aware that this can and will affect
    convergence at scale.
 
-.. clicmd:: ip pim keep-alive-timer (31-60000)
+.. clicmd:: ip pim keep-alive-timer (1-65535)
 
-   Modify the time out value for a S,G flow from 31-60000 seconds. 31 seconds
-   is chosen for a lower bound because some hardware platforms cannot see data
+   Modify the time out value for a S,G flow from 1-60000 seconds. If choosing
+   a value below 31 seconds be aware that some hardware platforms cannot see data
    flowing in better than 30 second chunks. This command is vrf aware, to
    configure for a vrf, enter the vrf submode.
 
-.. clicmd:: ip pim packets (1-100)
+.. clicmd:: ip pim packets (1-65535)
```

Comment:
YANG and command says 1-255, but doc still says 1-65535.

Reply:
> One nit left and I think it is good. I just can't understand why the OSPF test would fail.
> 
> The PIM test failure is either a intermittent failure or a race somewhere.

The OSPF failure is unrelated it is showing up elsewhere too.